environment.yml

```diff
@@ -14,3 +14,4 @@ dependencies:
   - biopet-scatterregions==0.1
   - tabix==0.2.5
   - vardict-java==1.5.4
+#  - somaticseq==2.8.1
```

Comment:
Yes. With containers, where do we record the used versions? That is the thing that I am going to miss with environment.yml

Reply:
We could keep the environment.yml as a list of versions we used to test.